packages/teams-js/src/public/imageAndVideo.ts

```diff
@@ -0,0 +1,316 @@
+     */
+    public format: FileFormat;
+
+    /**
+     * Size of the file in KB
+     */
+    public size: number;
+
+    /**
+     * MIME type. This can be used for constructing a dataUrl, if needed.
+     */
+    public mimeType: string;
+
+    /**
+     * Optional: Name of the file
+     */
+    public name?: string;
+  }
+
+  /**
+   * Specifies the type of GalleryFileType
+   */
+  export enum GalleryFileType {
+    /** Media type photo or image */
+    Image = 1,
+    /** Media type video. */
+    Video = 2,
+    /** Media type video and image. */
+    VideoAndImage = 3,
+  }
+
+  /**
+   * @hidden
+   * Hide from docs
+   * --------
+   * All properties common to Image and Video Props
+   */
+  interface GalleryInputs {
+    /**
+     * @hidden
+     * Optional; Lets the developer specify the media source
+     */
+    sources: Source;
+
+    /**
+     * @hidden
+     * Optional; Specify in which mode the camera will be opened.
+     * Default value is Photo
+     */
+    startMode?: CameraStartMode;
+
+    /**
+     * @hidden
+     * Optional; indicate if user is allowed to move between front and back camera
+     * Default value is true
+     */
+    cameraSwitcher?: boolean;
+    /**
+     * max limit of media allowed to be selected in one go, current max limit is 10 set by office lens.
+     */
+    maxMediaCount: number;
+  }
+  /**
+   *  All properties in ImageProps are optional and have default values in the platform
+   */
+  export interface ImageInputs extends GalleryInputs {
+    /**
+     * Optional; indicate if inking on the selected Image is allowed or not
+     * Default value is true
+     */
+    ink?: boolean;
+
+    /**
+     * Optional; indicate if putting text stickers on the selected Image is allowed or not
+     * Default value is true
+     */
+    textSticker?: boolean;
+
+    /**
+     * Optional; indicate if image filtering mode is enabled on the selected image
+     * Default value is false
+     */
+    enableFilter?: boolean;
+
+    /**
+     * Optional; Lets the developer specify the image output formats, more than one can be specified.
+     * Default value is Image.
+     */
+    imageOutputFormats?: ImageOutputFormats[];
+  }
+  /**
+   * The modes in which camera can be launched in select Media API
+   */
+  export enum CameraStartMode {
+    /** Photo mode. */
+    Photo = 1,
+    /** Document mode. */
+    Document = 2,
+    /** Whiteboard mode. */
+    Whiteboard = 3,
+    /** Business card mode. */
+    BusinessCard = 4,
+  }
+  /**
+   * Specifies the image source
+   */
+  export enum Source {
+    /** Image source is camera. */
+    Camera = 1,
+    /** Image source is gallery. */
+    Gallery = 2,
+  }
+  /**
+   * Specifies the image output formats.
+   */
+  export enum ImageOutputFormats {
+    /** Outputs image.  */
+    IMAGE = 1,
+    /** Outputs pdf. */
+    PDF = 2,
+  }
+  /**
+   * Output of getMedia API from platform
+   */
+  export interface MediaResult {
```

Comment:
This and MediaChunk aren't used as far as I can tell

Reply:
deleted, thanks for point it out!